server/k6_tests/script.js

```diff
@@ -0,0 +1,12 @@
+import { sleep } from 'k6';
+import http from 'k6/http';
+
+export const options = {
+  vus: 500,
+  duration: '1m30s',
+};
+
+export default function () {
+  http.get('http://localhost:3000/reviews?product_id=998999');
```

Comment:
Would recommend randomizing the product_id via something like Math.random()

Reply:
Could point David. I will do that!